spec/factories/settings.rb

```diff
@@ -0,0 +1 @@
+# Not sure what to put here
```

Comment:
Nothing to do here. Factories describe a model.  FactoryGirl is used to "create" objects to test against.  That way, we don't have to hit the database, which would make our tests SLOOOOOW